index.html

```diff
@@ -7,36 +7,35 @@
   <body>
     <script type="text/javascript">
 
-      const task = {
-        action:['机を片付ける','牛乳を買う','散歩する'],
-        genre:['掃除','買い物','運動']
-      };
+      const taskList = [
+        {action:'机を片付ける', genre:'掃除'},
+        {action:'牛乳を買う', genre:'買い物'},
+        {action:'散歩する', genre:'運動'},
+      ];
 
       const listing = () => {
```

Comment:
この名前だと「何を」Listing するのかがやや曖昧なので、以下のようにするとよいかと。
```suggestion
      const listTasks = () => {
```